<ui version="4.0" >
 <class>xmppWin</class>
 <widget class="QMainWindow" name="xmppWin" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>430</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>310</width>
    <height>430</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>310</width>
    <height>430</height>
   </size>
  </property>
  <property name="baseSize" >
   <size>
    <width>310</width>
    <height>430</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Kapture [Offline]</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="MouseRosterTableView" native="1" name="tableView" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>80</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <property name="showGrid" stdset="0" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="jid" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>10</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Full Jid</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>16</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text" >
     <string>Password</string>
    </property>
    <property name="frame" >
     <bool>true</bool>
    </property>
    <property name="echoMode" >
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="serverEdit" >
    <property name="geometry" >
     <rect>
      <x>0</x>
      <y>40</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="portEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>40</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text" >
     <string>5222</string>
    </property>
   </widget>
   <widget class="QLabel" name="tlsIconLabel" >
    <property name="geometry" >
     <rect>
      <x>250</x>
      <y>40</y>
      <width>32</width>
      <height>32</height>
     </rect>
    </property>
    <property name="minimumSize" >
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="maximumSize" >
     <size>
      <width>32</width>
      <height>32</height>
     </size>
    </property>
    <property name="text" >
     <string/>
    </property>
   </widget>
   <widget class="QComboBox" name="profilesComboBox" >
    <property name="geometry" >
     <rect>
      <x>30</x>
      <y>10</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="configBtn" >
    <property name="geometry" >
     <rect>
      <x>50</x>
      <y>40</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text" >
     <string>Configuration</string>
    </property>
   </widget>
   <widget class="QComboBox" name="statusBox" >
    <property name="geometry" >
     <rect>
      <x>170</x>
      <y>10</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text" >
      <string>Online</string>
     </property>
    </item>
    <item>
     <property name="text" >
      <string>Offline</string>
     </property>
    </item>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MouseRosterTableView</class>
   <extends>QWidget</extends>
   <header>mouserostertableview.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>jid</tabstop>
  <tabstop>password</tabstop>
  <tabstop>tableView</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
